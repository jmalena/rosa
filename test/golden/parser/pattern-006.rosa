--| Parenthesized variable pattern should fail.

id : x -> x
id (x) = x