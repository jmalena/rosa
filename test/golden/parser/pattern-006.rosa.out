This is wrong error message.